src/routes/boot.ts

```diff
@@ -74,13 +88,15 @@ export default async function (fastify: FastifyInstance): Promise<void> {
         ]),
         notifications: { unreadNotificationsCount },
         squads,
+        features,
       });
     }
     return res.send({
       alerts: ALERTS_DEFAULT,
       settings: SETTINGS_DEFAULT,
       notifications: { unreadNotificationsCount: 0 },
       squads: [],
+      features: [],
```

Comment:
Should be an empty object not array
```suggestion
      features: {},
```